package.json

```diff
@@ -24,6 +25,8 @@
   "scripts": {
     "test": "tap -j 4 test/parallel/*.js test/ours/*.js",
     "ci": "TAP=1 tap test/parallel/*.js test/ours/*.js | tee test.tap",
+    "test-browsers": "airtap --sauce-connect --no-coverage -- test/browser.js",
```

Comment:
With latest `airtap` (0.0.9) you no longer need `--no-coverage`. Collecting code coverage became opt-in.